src/Types/VersionedModuleSource.cs

```diff
@@ -14,6 +16,17 @@ public abstract record VersionedModuleSource
     public byte[] Source { get; }
     private readonly Lazy<Module> _module;
 
+    internal const uint MaxLength = 8 * 65536;
```

Comment:
**Request change:** The max size in bytes is 65535, so I don't think we should multiply by 8 here. Also it seems to be off by one.
```suggestion
    internal const uint MaxLength = 65535;
```

Reply:
I think this was supposed to be `2*sizeof(ushort) + 65535` for the versioning. But maybe I'm misunderstanding the `MaxLength` here. I'll check some documentation/rust code.

Reply:
No that's wrong, I got the `8 * 65536` from here:
https://github.com/Concordium/concordium-base/blob/6d9ad3500d0150062d8596715f6920ff33678c89/rust-src/concordium_base/src/constants.rs#L19

Reply:
My mistake, I was using the max size of the parameter, not the module. So you were correct all along `512 kB = 8 * 65536`.